### PR TITLE
Add android-arm64 to Travis

### PR DESCRIPTION
**Pull request by JonasVautherin**
If that works (i.e. including testing on an Android device), I will add android-arm and deploy that as an artifact.

### CODE REVIEW

**Comment by JonasVautherin** [comment]
It works! :tada:

File: .travis.yml [hunk 1]
```diff
@@ -205,86 +233,52 @@ after_success:
...
-      if [[ "${BUILD_TARGET}" = "linux-arm64" ]]; then
-          mv build/linux-arm64/install/bin/mavsdk_server build/linux-arm64/install/bin/mavsdk_server_linux-arm64;
-      fi;
-      if [[ "${BUILD_TARGET}" = "osx_build" ]]; then
-          mv build/release/install/bin/mavsdk_server build/release/install/bin/mavsdk_server_macos;
-      fi;
-      if [[ "${BUILD_TARGET}" = "windows" ]]; then
-          mv build/release/install/bin/mavsdk_server_bin.exe build/release/install/bin/mavsdk_server_win32.exe;
-      fi;
+    export BEFORE_DEPLOY_RUN=1;
```
Comment (julianoes) [inline .travis.yml]: What is this actually for?

File: .travis.yml [hunk 2]
```diff
@@ -205,86 +233,52 @@ after_success:
...
-    condition: ${BUILD_TARGET} = osx_build
-    repo: mavlink/MAVSDK
-    tags: true
-- provider: releases
-  skip_cleanup: true
-  api_key:
-      secure: hBX3pFWNZiDbz4yKnOjhLg3QS9Ubn1XePxSeIt2Btq5GzbomOPDCgpIFijBppliwj9oKc302EMnZSg2QWeAzFKn9UnmIflJ0E4iymYgwWdTJv+bSnYALJEmO8F6gF9FgRlPk8FCtZiECoTsa75w5TrEZKZpFpmzVYRiDu0eo6sEjW7UJPC0A2KSTXLrBCHSIZy/iasbGmuur4brG7NO0QdMOvDXvhsYfkXDRJFMTtTHvLiKJcqiunPfqARzf1H4x4iczRYscKu5Vn8Kmw3NANGkcIDvEj4ooih831EXxACRZw0VgycgNHOKRXKC9pZ4hLQMon+jxpQX+X8k/K5161oEkF/gCVKyFb31Pk/4Uwe81p1GJY2lAC7MDUxA98RKXhdvVYF2Cp44+IbF0YVoWRUtVAhknXRQ3Weg25kyVSu83q2nN2nZq2qGTnpNIbdN56s/F+uaFtipGEh+vmiv8rNUz+Z5MFrY2FQaSvBTFw9K4tNs9uc+VQd1bE7X5wh0yywEqUEw2nzqTB2xR+OubygUASbk2GLNdc254P0lrzCHbNM62Y7sRX06CM7hPlwhELEkVtUXZWJ0KuhQyLvRh3aPJ3Jj30EswTt/FGT1gzSP1FjjHBRZCK4P2D2rwJ5TMn2JrZKfPxmEd3kVmn6h80+gBbKgonGmZspd2SvPEI5g=
-  file_glob: true
-  file:
-  - "*.deb"
```
Comment (julianoes) [inline .travis.yml]: Hang on, what happened to this!

File: .travis.yml (hunk 1 above)
Comment (JonasVautherin) [reply .travis.yml]: I don't know...

File: .travis.yml (hunk 2 above)
Comment (JonasVautherin) [reply .travis.yml]: Oh, good catch, sorry about that. Let me try to add that back. For the others, it seems like it works, though

Comment (julianoes) [reply .travis.yml]: ok.

**Pull request merged**